src/pybind/mgr/cephadm/module.py

```diff
@@ -104,7 +104,7 @@ def os_exit_noop(status: int) -> None:
 
 
 # Default container images -----------------------------------------------------
-DEFAULT_IMAGE = 'quay.io/ceph/ceph'
+DEFAULT_IMAGE = 'quay.io/athakkar/nfs'
```

Comment:
Will need to be removed before merge or run thru teuthology.

Reply:
Yes! I'll remove these changes.